diff --git a/CMakeLists.txt b/CMakeLists.txt
index eda7acae..a84fd7f5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -44,6 +44,7 @@ message(STATUS "Git revision: ${HERMES_REVISION}")
 option(HERMES_BUILD_BOUT "Build BOUT++ in external/BOUT-dev" ON)
 if(HERMES_BUILD_BOUT)
   set(BOUT_BUILD_EXAMPLES, OFF) # Don't create example makefiles
+  set(BOUT_ENABLE_PYTHON OFF)
   set(HERMES_BOUT_SRC
       "external/BOUT-dev"
       CACHE STRING "Directory of BOUT++ dir")
